### PR TITLE
Update the Readme.md #1857

### LINKED ISSUE #1857: Update the Readme.md

### PR DESCRIPTION
**Pull request by CrypticRevenger**
#1857 
updated in correcting the grammar, verbs, articles , spellings, preposition ,pronoun.

Fixes #1857

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,100 +67,100 @@ mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
...
-<br>`--installer` argument can be added to build installers for particular OS. NOTE: MacOS Installer can be built with MacOS only.
+directory. Running this task on macOS or Linux requires Wine since it's needed to set the icon
+for the Windows app. If you don't have Wine installed you can create a release by running the **release-only-Linux** task.
+<br>`--installer` argument can be added to build installers for a particular OS. NOTE: MacOS Installer can be built with MacOS only.
 
 To build a specific release, use the command `release --platform="win64"` for example.
 
 ### Running with debug | Inspector
 
-To be able to open Inspector, you will need SDK flavours of NW.js
```
Comment (MrD-RC) [inline README.md:124]: Flavours is the correct spelling. Please revert this change.

File: README.md [hunk 2]
```diff
@@ -67,100 +67,100 @@ mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
...
 	1. Enter MapProxy service layer (inav_layer if configured from MAPPROXY.md)
-1. Once completed, you can zoom in on area you will be flying in while connected to the internet in either GPS or Mission Control tab to save the cache for offline use
+1. Once completed, you can zoom in on the area you will be flying in while connected to the internet in either the GPS or Mission Control tab to save the cache for offline use
 
 ## Font Customisation
 
-INAV provides the font images so that custom fonts can be created for your personal preference. This is the case for both analogue and digital fonts. The resources can be found in the [osd](/resources/osd) folder. Within the **analogue** and **digital** subfolders, you will find information on compiling your own fonts. There is also an [INAV Character Map](/resources/osd/INAV%20Character%20Map.md) document. This contains previews of all the character images in the fonts, and the appropriate variable names within the firmware and Configurator. There are tools for compiling the [analogue](https://github.com/fiam/max7456tool) and [digital](https://github.com/MrD-RC/hdosd-font-tool) fonts. New font submissions via pull requests are welcome.
```
Comment (MrD-RC) [inline README.md:163]: I think this one is fine as is.

**Review by MrD-RC — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -67,100 +67,100 @@ mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
...
 #### Notes
 
-On some Linux distros, you may be missing `libatomic`, a `NW.JS` (specially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
+On some Linux distros, you may be missing `libatomic`, an `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
 
 * Arch Linux: `sudo pacman -S --needed libatomic_ops`
 * Debian / Ubuntu: `sudo apt install libatomic1`
 * Fedora: `sudo dnf install libatomic`
 
-1. Don't forget to add your user into dialout group "sudo usermod -aG dialout YOUR_USERNAME" for serial access
```
Comment (MrD-RC) [inline README.md:87]: It may be worth leaving `dialout` as is. Because it may refer to the name of an actual group in Linux.

File: README.md (hunk 1 above)
Comment (MrD-RC) [reply README.md:124]: Sorry, no need for the uppercase F. That was only there as it was the start of a sentence.

File: README.md (hunk 3 above)
Comment (CrypticRevenger) [reply README.md:87]: ok

**Comment by CrypticRevenger** [comment]
@MrD-RC  please review the pr.

File: README.md [hunk 4]
```diff
@@ -67,55 +67,55 @@ mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
...
 10. Make the following files executable:
    * inav-configurator `chmod +x /opt/inav/inav-configurator/inav-configurator`
    * (5.0.0+) chrome_crashpad_handler `chmod +x /opt/inav/inav-configurator/chrome_crashpad_handler`
-11. Run INAV Configurator app from unpacked folder `/opt/inav/inav-configurator/inav-configurator`
+11. Run the INAV Configurator app from the unpacked folder `/opt/inav/inav-configurator/inav-configurator`
 
 #### Notes
 
-On some Linux distros, you may be missing `libatomic`, a `NW.JS` (specially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
+On some Linux distros, you may be missing `libatomic`, an `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
```
Comment (MrD-RC) [inline README.md:81]: I think this would be better as:

missing `libatomic` and/or `NW.JS` (especially `libnode.so`) dependencies.

**Review by MrD-RC — CHANGES_REQUESTED** [review]

**Comment by MrD-RC** [comment]
Spotted another change.

File: README.md [hunk 5]
```diff
@@ -78,7 +78,7 @@ sudo mv inav-configurator.desktop /usr/share/applications/
 
 #### Notes
 
-On some Linux distros, you may be missing `libatomic`, an `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
+On some Linux distros, you may be missing `libatomic`or `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
```
Comment (MrD-RC) [inline README.md:81]: I think is should be **and/or** because it could be both, one, or none. Also, **dependency** should be **dependencies** because it should be plural.

File: README.md (hunk 2 above)
Comment (MrD-RC) [reply README.md:163]: This one also still needs resolution.

**Comment by CrypticRevenger** [comment]
@MrD-RC I think there should  be "Z" in  "Font customization" .
and the previous issue resolved

**Comment by MrD-RC** [comment]
> @MrD-RC I think there should be "Z" in "Font customisation" . and the previous issue resolved

Definitely no ‘z’ in Customisation. Z are in the American spelling, not English spelling.

**Review by MrD-RC — APPROVED** [review]

**Comment by CrypticRevenger** [comment]
@MrD-RC sir, can you please merge this issue,
That will be very helpful for me.
Thank you.

**Comment by MrD-RC** [comment]
Thanks for doing this.

Just one thing to note is that some of the words may look incorrect. For example `dialout`. But they are technically correct because they are referencing something in code or an OS. A lot of the time, they will likely be in a single or triple single-quote block. But they could be anywhere. To be honest, I was unsure about dialup. Until I saw it in the code to be entered in to Terminal on the next line.
